lib/testing/src/lib/process-services/pages/form-fields.page.ts

```diff
@@ -138,6 +149,11 @@ export class FormFieldsPage {
         await BrowserVisibility.waitUntilElementIsVisible(widgetReadOnly);
     }
 
+    async isFormFieldEnabled(formFieldId: string): Promise<boolean> {
+        const formField = element(by.css(`input[id=${formFieldId}]`));
```

Comment:
`input[id=${formFieldId}]` can be simplified into `input#${formFieldId}`

Reply:
Also, since an `id` is unique, you should be able to use just 

    by.id(`${formFieldId}`)

Reply:
Done. Thanks